agents-manage-ui/src/app/[tenantId]/projects/[projectId]/agents/[agentId]/loading.tsx

```diff
@@ -2,28 +2,35 @@ import type { FC } from 'react';
 import { BodyTemplate } from '@/components/layout/body-template';
 import { Skeleton } from '@/components/ui/skeleton';
 
-const AgentLoadingSkeleton: FC = () => {
+export const AgentLoadingSkeleton: FC = () => {
   return (
-    <BodyTemplate breadcrumbs={[{ label: 'Agents' }, { label: '' }]}>
-      <div className="flex h-screen bg-muted/20 dark:bg-background p-4">
-        <div className="w-52">
-          <div className="space-y-2">
-            <Skeleton className="h-12 w-full rounded-lg" />
-            <Skeleton className="h-12 w-full rounded-lg" />
-          </div>
+    <div className="flex h-screen bg-muted/20 dark:bg-background p-4">
+      <div className="flex flex-col gap-2" style={{ width: 160 }}>
+        <Skeleton style={{ height: 38 }} />
+        <Skeleton style={{ height: 38 }} />
+        <Skeleton style={{ height: 38 }} />
+        <Skeleton style={{ height: 38 }} />
+        <Skeleton style={{ height: 38 }} />
+      </div>
+      <div className="flex-1 flex flex-col">
+        <div className="flex justify-end gap-2 h-9">
+          <Skeleton style={{ width: 84 }} />
+          <Skeleton style={{ width: 100 }} />
+          <Skeleton style={{ width: 127 }} />
+          <Skeleton style={{ width: 168 }} />
         </div>
-        <div className="flex-1 flex flex-col">
-          <div className="flex justify-end gap-3">
-            <Skeleton className="h-9 w-20" />
-            <Skeleton className="h-9 w-16" />
-          </div>
-          <div className="w-64 absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2">
-            <Skeleton className="h-24 w-full rounded-lg" />
-          </div>
+        <div className="w-64 absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2">
+          <Skeleton className="h-36 w-full rounded-lg" />
         </div>
       </div>
-    </BodyTemplate>
```

Comment:
enhance skeleton based on current graph
<img width="1472" height="699" alt="Screenshot 2025-12-18 at 00 48 44" src="https://github.com/user-attachments/assets/a08f8c88-b6e0-47cd-ba43-55695aee4f90" />

Reply:
Came across these two options, neither are really new packages, but they are both new to me.

Option 1:
https://www.npmjs.com/package/react-loading-skeleton
The suggestion is that each component has a Skeleton baked into it and there is a loading state that will transition between the manually defined Skeleton or the Actual component

Option 2: (a little bit newer), but requires some investigation before we commit to it
https://www.npmjs.com/package/react-skeletonify
Uses some SkeletonProvider to automatically define skeletons for components without having to manually define them.
I'm not aware of any of the performance implications of this package yet, we should consider if the package handles static/dynamic pages, SSE, or other such potential edge cases.